main/dev/css/components/avatar/avatar.html

```diff
@@ -42,43 +42,81 @@ <h2 class="pad-10">components</h2>
                 </ul>
             </div>
             <div class="docs pad-10">
-                <div class="avatarRound">
-                    <img class="avatar-round" src="../../../asserts/avatarMale.jpg" alt="avatar">
+                <h1>Avatar</h1>
+                <p>Avatar components of three different variants circle with image, square with image and circle
+                    with text and 5 different sizes can be found. Use the class avatar-round,avatar-square,avatar-text
+                    for corresponding avatar component and
+                    classes avatar-xs avatar-sm avatar-md avatar-lg avatar-xl for avatars of different sizes</p>
 
-                    <img class="avatar-round avatar-xs" src="../../../asserts/avatarMale.jpg" alt="avatar">
+                <div class="items-center">
+                    <h2>Round-Avatar</h2>
+                    <div class="avatarRound box">
+                        <img class="avatar-round avatar-xs" src="../../../asserts/avatarMale.jpg" alt="avatar">
```

Comment:
use valid avatar alt names..
I think we can't use same alt name everywhere!
it affects accessibility